client/app/js/annotator-madison.js

```diff
@@ -280,15 +279,10 @@ $.extend(Annotator.Plugin.Madison.prototype, new Annotator.Plugin(), {
   addNoteLink: function (field, annotation) {
 
     //Add link to annotation
-    var annotationLink = $('<a></a>').attr('href', this.options.path + '#' + annotation.link).text('Copy Annotation Link').addClass('annotation-permalink');
-    var noteLink = $('<div class="annotation-link"></div>');
     var linkPath = this.options.origin + this.options.path + '#' + annotation.link;
```

Comment:
With `this.options.origin` at the beginning here, it results in the origin being in there twice. Removing that makes these functional again.